openapi3filter/req_resp_decoder.go

```diff
@@ -616,9 +616,14 @@ func (d *urlValuesDecoder) parseValue(v string, schema *openapi3.SchemaRef) (int
 	}
 
 	return parsePrimitive(v, schema)
-
 }
 
+const (
+	// these should not conflict with anything
+	urlArrayDelimiter     = "\x1F"
+	urlObjectKeyDelimiter = "\x1F"
```

Comment:
These are identical. Typo? If not, let's just introduce a single const?

Reply:
Merged into one delimiter